packages/Python/lldbsuite/test/lang/swift/expression/equality_operators/Makefile

```diff
@@ -1,20 +1,14 @@
-LEVEL = ../../../../make
-
 EXE=three
 SWIFT_SOURCES=three.swift
-LD_EXTRAS=-L$(shell pwd) -lfooey
-SWIFTFLAGS_EXTRAS= -I$(shell pwd)
+LD_EXTRAS=-L. -lfooey
```

Comment:
One reason to prefer `$(BUILDDIR)` over `.` is that the resulting compiler invocations can be copied & pasted into a terminal without having to cd into the build dir first.

In any case, it would be nice to pick one and stick with it.